images/runtime/php/7.0/docker-php-ext-enable

```diff
@@ -0,0 +1,114 @@
+#!/bin/sh
+set -e
```

Comment:
did you verify manually if this works for php 7?

Reply:
The Nightly build passed after the changes in this PR https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=3212379